bundles/core/org.eclipse.smarthome.core.library.test/src/test/java/org/eclipse/smarthome/core/library/items/DimmerItemTest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.library.items;
+
+import static org.junit.Assert.assertEquals;
+
+import java.math.BigDecimal;
+
+import org.eclipse.smarthome.core.items.Item;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.types.State;
+import org.junit.Test;
+
+/**
+ * @author Chris Jackson - Initial contribution
+ * @author Markus Rathgeb - Add more tests
+ */
+public class DimmerItemTest {
+
+    private static DimmerItem createDimmerItem(final State state) {
+        final DimmerItem item = new DimmerItem("Test");
+        item.setState(state);
+        return item;
+    }
+
+    private static BigDecimal getState(final Item item, Class<? extends State> typeClass) {
+        final State state = item.getStateAs(typeClass);
+        final String str = state.toString();
+        final BigDecimal result = new BigDecimal(str);
+        return result;
+    }
+
+    @Test
+    public void GetAsPercentFromPercent() {
```

Comment:
Did you name the tests on with a capital letter on purpose? Looks rather unusual to me...